src/main/java/com/example/server/localization/LocalizationController.java

```diff
@@ -32,6 +37,20 @@ public String createLocalization(String json) {
             return "{\"error\": \"Internal server error\"}"; // http 500
         }
     }
+    // GET /localizations
+    public String getLocalizations() throws JsonProcessingException {
+        try{
+            List<Localization> localizations = getLocalizationsService.getAllLocalizations();
+            return objectMapper.writeValueAsString(localizations
```

Comment:
try this line `objectMapper.writeValueAsString(localizations);

Reply:
super, zadziałało